src/main/target/common_pre.h

```diff
@@ -199,6 +199,7 @@
 #define USE_ITERM_RELAX
 #define USE_DYN_LPF
 #define USE_INTEGRATED_YAW_CONTROL
+#define USE_THRUST_LINEARIZATION
```

Comment:
SIRINFPV is now overflowing as well. :-(

Reply:
:(. Ok turned it off there, too.

Reply:
Welcome to the world of ~pain~ playing whack-a-mole with features on F3.